lib/backend-create.js

```diff
@@ -27,52 +27,60 @@ const nameManager = require('./utils/awsmobilejs-name-manager.js')
+                    default: mobileProjectName
+                }
+            ]).then(function (answers) {
+                if(answers.mobileProjectName){
+                    createProject(projectInfo, awsDetails, answers.mobileProjectName, options, callback)
                 }
+            })
+        }
+    })
+}
+
+function createProject(projectInfo, awsDetails, mobileProjectName, options, callback){
 
-                let spinner = ora('creating backend awsmobile project ' + param.name)
-                spinner.start()
-                mobile.createProject(param, function(err,data){
-                    spinner.stop()
+    let mobile = awsClient.Mobile(awsDetails)
+
+    let backendContents = getBackendContents(options)
+    let param = {
+        name: mobileProjectName,
+        region: awsDetails.config.region,
+        contents: backendContents
+    }
+
+    let spinner = ora('creating backend awsmobile project ' + param.name)
+    spinner.start()
+    mobile.createProject(param, function(err,data){
+        spinner.stop()
+        console.log()
+        if(err){
+            console.log(chalk.red('backend awsmobile project creation error'))
+            awsExceptionHandler.handleMobileException(err)
+        }else if(data && data.details){
+            backendWaitLogic.wait(data.details, awsDetails, (err, backendDetails) => {
+                if(!err){ 
+                    console.log('Successfully created AWS Mobile Hub project: ' + chalk.blue(data.details.name))
                     console.log()
-                    if(err){
-                        console.log(chalk.red('backend awsmobile project creation error'))
-                        awsExceptionHandler.handleMobileException(err)
-                    }else if(data && data.details){
-                        backendWaitLogic.wait(data.details, awsDetails, (err, backendDetails) => {
-                            if(!err){ 
-                                console.log('Successfully created AWS Mobile Hub project: ' + chalk.blue(data.details.name))
-                                console.log()
-                                backendInfoManager.syncCurrentBackendInfo(projectInfo, backendDetails, awsDetails, options.syncToDevFlag, function(){
-                                    if(callback){
-                                        callback()
-                                    }
-                                })
-                            }
-                        })
-                    }else{
-                        console.log(chalk.red('something went wrong'))
-                    }
-                })
-            } 
-        })
+                    backendInfoManager.syncCurrentBackendInfo(projectInfo, backendDetails, awsDetails, options.syncToDevFlag, function(){
+                        if(callback){
+                            callback()
+                        }
+                    })
```

Comment:
There is no message if backendWaitLogic fails

Reply:
the backend-wait-loic.js file has the implementation, when the wait log has failed, proper error messages are printed out on the console. 
https://github.com/aws/awsmobile-cli/blob/39bf9c7c718a1090ae99afee271bee55cd5afed7/lib/backend-wait-logic.js#L65